docs/src/core-concepts/codeact.md

```diff
@@ -255,6 +255,6 @@ Planned improvements:
 
 ## Next Steps
 
-- Explore [ReAct agents](./react.md) for tool-based approaches
```

Comment:
@vicentereig The `react.md` file doesn't exist, I presume you wanted to create separate docs for it. For now there's a section in `predictors`, which is linked here in this PR.